crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -10,6 +10,10 @@ pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt252>> {
     let re_string = Regex::new(r#"(?s)Execution failed\. Failure reason: "(.*?)"\."#)
         .expect("Could not create string panic_data matching regex");
 
+    let re_entry_point =
+        Regex::new(r"Entry point EntryPointSelector\((0x[0-9a-fA-F]+)\) not found in contract\.")
+            .unwrap();
+
```

Comment:
Could you please tell where this regex is taken from? From what I see in [spec](https://github.com/starkware-libs/starknet-specs/blob/6d88b7399f56260ece3821c71f9ce53ec55f830b/api/starknet_api_openrpc.json#L2455), entry point selector points to felt which has different regex 🤔 .

Also, if it's from spec - we should include a link in the comment as reference.

Reply:
Upadated